src/libraries/System.Data.Common/tests/System/Data/SqlTypes/SqlXmlTest.cs

```diff
@@ -119,6 +96,82 @@ public void SqlXml_fromXmlReader_CreateReaderTest()
             Assert.Equal(xmlStr, xrdr.ReadOuterXml());
         }
 
+        [Theory]
+        [InlineData("element_whitespace-text.xml")]
+        [InlineData("root_qname.xml")]
+        [InlineData("sample_ecommerce.xml")]
+        [InlineData("sql_batch_request.xml")]
+        [InlineData("sql_batch_response.xml")]
+        [InlineData("sql_datatypes-1.xml")]
+        [InlineData("sql_datatypes-2.xml")]
+        [InlineData("sql_datatypes-3.xml")]
+        [InlineData("xmlns-1.xml")]
+        [InlineData("xmlns-2.xml")]
+        [InlineData("xmlns-3.xml")]
+        [InlineData("xmlns-4.xml")]
+        [InlineData("comments_pis.xml")]
+        [InlineData("element_content_growth.xml")]
+        [InlineData("element_nested-1.xml")]
+        [InlineData("element_nested-2.xml")]
+        [InlineData("element_nested-3.xml")]
+        [InlineData("element_single.xml")]
+        [InlineData("element_stack_growth.xml")]
+        [InlineData("element_tagname_growth.xml")]
+        [InlineData("element_types.xml")]
+        [InlineData("element_whitespace-modes.xml")]
+        public void SqlXml_fromXmlReader_TextXml(string filename)
```

Comment:
Nit:
```suggestion
        public void SqlXml_FromXmlReader_TextXml(string filename)
```

Reply:
I was following the existing capitalization pattern in the test file, but I'll clean them all up instead.